vignettes/efficient-calculations.Rmd

```diff
@@ -100,64 +100,81 @@ results <- runSimulations(simulations = list(sim100, sim200, sim300, sim400, sim
+This implementation is also fairly straight forward. We load the simulation multiple times from the same file (note the usage of `loadFromCache = FALSE` to ensure that we always get a new instance of a simulation and not the same instance) and we set the dose. Then, we call the `runSimulations` function with the list of simulations. The engine will run these simulations in parallel, which means that the overall execution time should be slightly more than 1h, as each run would be executed **at the same time**.
 
 
-## Running one or more simulations multiple times in a loop
+## Optimized loading for running simulations with pre-defined set of variable parameters
 
-This is a more advanced use case, typically when implementing some kind of optimization or sensitivity algorithms. The ospsuite-r package introduces the concept of a **simulation batch**. With a simulation batch, the variable parameters (or initial values) of a simulation to vary between each run are specified explicitely. This allows the simulation engine to speed up the system significantly as some equations can be rewritten and simplified. A simulation batch also keeps the simplified simulation in memory. That means that running a simulation again with a new set of values will be much faster, as the initialization phase (4) is only done if required and not for every run. 
+This is a more advanced use case, typically when implementing some kind of optimization or sensitivity algorithms. The ospsuite-r package introduces the concept of a **simulation batch**. With a simulation batch, the variable parameters (or initial values) of a simulation to vary between each run are specified explicitly during the creation of the simulation batch. This allows the simulation engine to speed up the system significantly as some equations can be rewritten and simplified. A simulation batch also keeps the simplified simulation in memory. That means that running a simulation again with a new set of values will be much faster, as the initialization phase (4) is only done if required and not for every run. 
 
-Consider the following implementation:
-```{r BatchRun, eval=TRUE}
+Consider the following implementation. We first create two simulation batches for the same model
+and define the parameter values that can be changed between runs. Note that, as opposed to 
+the `Simulation` object, we do not have access to any parameter/species values nor can we change
+the outputs or simulation time.
+```{r BatchRunInit, eval=TRUE}
 library(ospsuite)
 simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
 
-# We create two different instances of the same simulation to illustrate the usage.
+# We load the simulation for which the batches will be created
 sim1 <- loadSimulation(simFilePath, loadFromCache = FALSE)

```

Comment:
Actually, no two sims needed here.

Reply:
Yes this was just to say that you can have multiple simulations and not only one simulation